File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/java/python/p03826.py
Here is the Python equivalent of the Java code:

```python
def main():
    a, b, c, d = map(int, input().split())
    a = a * b
    b = c * d
    print(b if a < b else a)

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads four integers from the user, multiplies the first two and the last two, and then prints the larger of the two results.

File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/python/java/p03826.java
Here is the Java equivalent of the Python code you provided:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        int d = scanner.nextInt();
        
        int s = a * b;
        int t = c * d;
        
        if (s > t) {
            System.out.println(s);
        } else {
            System.out.println(t);
        }
    }
}
```

This Java program does the same thing as your Python code. It reads four integers from the user, calculates the product of the first two and the last two, and then prints the larger of the two products.
